packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -489,20 +489,24 @@ async function handler(
     bookerEmail,
   });
 
-  // If a request already exists for the timeslot, return that request
-  if (!isConfirmedByDefault && !userReschedulingIsOwner) {
-    const requestedBooking = await getBookingRequest({
+  // For unconfirmed bookings or round robin bookings with the same attendee and timeslot, return the original booking
```

Comment:
Right now unconfirmed bookings and round robin events let an attendee book the same timeslot multiple times. Seats has it's own logic to handle that scenario.